### PR TITLE
gcp cloudtasks added

### PR DESCRIPTION
**Pull request by GauravChawda**

### CODE REVIEW

File: cartography/intel/gcp/cloudtasks.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        else:
+            raise
+
+
+@timeit
+def transform_cloudtasks_locations(locations: List[Dict]) -> List[str]:
+    transformed_locations = []
+    for location in locations:
+        location_id = location.get('locationId', None)
+        if not location_id:
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/cloudtasks.py:49]: check for `not` here. is `not` require here?

File: cartography/intel/gcp/resources.py [hunk 2]
```diff
@@ -23,8 +26,9 @@
 from . import cloudcdn
 from . import loadbalancer
 from . import bigquery
-from . import spanner
-from . import pubsublite
+<< << << < HEAD
+== == == =
+>>>>>> > ce5d7086e0711afa1dad6457719e58fef723f78b
```
Comment (ganeshjagdale9545) [inline cartography/intel/gcp/resources.py:31]: Remove this

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
few minor changes needed

**Review by mpurusottamc — APPROVED** [review]
looks good to me.

**Pull request merged**